pyanaconda/modules/network/installation.py

```diff
@@ -290,10 +291,11 @@ def _copy_global_dns_config(self, root):
         :param root: path to the root of the target system
         :type root: str
         """
-        src = self.NM_GLOBAL_DNS_RUNTIME_CONFIG
-        dst = os.path.normpath(root + self.NM_GLOBAL_DNS_CONFIG)
-        if os.path.isfile(src):
-            shutil.copy(src, dst)
+        for file in self.NM_GLOBAL_DNS_CONFIG_FILES:
+            src = os.path.join(self.NM_GLOBAL_DNS_RUNTIME_CONFIG_DIR, file)
+            dst = os.path.normpath(root + os.path.join(self.NM_GLOBAL_DNS_CONFIG_DIR, file))
+            if os.path.isfile(src):
+                shutil.copy(src, dst)
```

Comment:
Shouldn't we set permission of the files?

Reply:
I think we don't need to. We are not doing in any module and it is working as expected.